packages/transport/src/core/Transport.ts

```diff
@@ -6,6 +6,7 @@ import { SodiumWrapper } from "@nmshd/crypto";
 import { AgentOptions } from "http";
 import { AgentOptions as HTTPSAgentOptions } from "https";
 import _ from "lodash";
+import { PeerCertificate, checkServerIdentity } from "tls";
```

Comment:
I think this will kill App builds, I have a feeling this PR should be in the Connector.

Reply:
> I think this will kill App builds

How that?

Reply:
the tls package is only available in nodejs and not in browsers